src/main/java/unicash/logic/parser/UniCashParser.java

```diff
@@ -77,13 +78,13 @@ public Command parseCommand(String userInput) throws ParseException {
             return new GetTotalExpenditureCommandParser().parse(arguments);
 
         case ClearTransactionsCommand.COMMAND_WORD:
-            return new ClearTransactionsCommand();
+            return new ClearTransactionsCommandParser().parse(arguments);
```

Comment:
Was wondering why is there new parsers for these commands and a new restriction to have no user input at all? I think `Parsers` are only apt when there's arguments to process, if there aren't any, then we don't need to impose restrictions or create new `Parsers` that just effectively create a new `Command` that we had already done

Reply:
https://github.com/AY2324S1-CS2103-T16-3/tp/pull/130#issuecomment-1785450343